ivy/functional/frontends/torch/linalg.py

```diff
@@ -170,3 +170,7 @@ def eig(input, *, out=None):
 @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
 def solve(input, other, *, out=None):
     return ivy.solve(input, other, out=out)
+ 
+@to_ivy_arrays_and_back
+def vander(A, *, increasing=False, out=None):
```

Comment:
variable names and keywords of the functions should match those in torch, i.e. `x`, `N=None`, `increasing=False`